tests/snippets/whats_left_to_implement.py

```diff
@@ -1,873 +0,0 @@
-import platform
```

Comment:
I'd rather keep the generated file checked in into GIT. Please add a note that the file is generated automatically by the generator script, and check it into GIT so that people can run tests straight away. I think it is good to keep the generated file, as it gives us a common reference. If this file is generated on the fly, it depends upon installed cpython what will be generated, so results vary between PC's.

Reply:
On the other hand, maybe it is also fine to leave it out of GIT.

Reply:
would it be better to insert a warning in the generator or in the generated file?

Reply:
In the generated file, a warning should be put that the file was automatically generated, and that manual editing is not handy.